gateware/vga/dma.py

```diff
@@ -0,0 +1,134 @@
+from migen.fhdl.std import *
```

Comment:
Sigh, how did I miss that. We are using DMA from `gateware/hdmi_in/dma.py` itself. This file probably got copied initially and was never removed. 
![screenshot280](https://cloud.githubusercontent.com/assets/1108318/14052233/9dccfe50-f2ee-11e5-9ca1-993a442a86aa.png)